labgrid/driver/power/gude8031.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+from ..exception import ExecutionError
+
+# Driver has been tested with:
+# Gude Expert Power Control 8031()
+
+# Components Parameter is static, defines how to interact whith power controller
+
+PORT = 80
+
+def power_set(host, port, index, value):
+    index = int(index)
+    assert 1 <= index <= 8
+    # access the web interface...
+    value = 1 if value else 0
+    r = requests.get(
+        "http://{}:{}//status.json?components=769&cmd=1&p={}&s={}".format(host, port, index, value)
+    )
+    r.raise_for_status

```

Comment:
This is missing the function call:
```suggestion
    r.raise_for_status()
```